public.yaml

```diff
@@ -72,47 +121,46 @@ spec:
     - name: VELUM_DB_PASSWORD
       value: "salt"
     - name: VELUM_DB_SOCKET
-      value: /var/run/mysql/mysql.sock
+      value: "/var/run/mysql/mysql.sock"
     - name: VELUM_SALT_HOST
       value: "127.0.0.1"
     - name: VELUM_SALT_PORT
       value: "8000"
     - name: VELUM_SALT_USER
-      value: saltapi
+      value: "saltapi"
     - name: VELUM_SALT_PASSWORD
-      value: saltapi
+      value: "saltapi"
     volumeMounts:
       - mountPath: /var/run/mysql
         name: mariadb-unix-socket
       - mountPath: /var/lib/misc/velum-secrets
         name: velum-secrets
-    command: ["/bin/sh","-c"]
-    args: ["bin/init"]
+    command: ["/bin/sh", "-c", "bundle exec rails s -b 0.0.0.0 -p $VELUM_PORT --pid /tmp/puma.pid Puma"]
```

Comment:
Improving `bin/init` would help here too... we could do something like `bin/init --skip-initialization` or something like that... that way we avoid duplicating commands like this one...